changelog.txt

```diff
@@ -1,5 +1,24 @@
 *** Changelog ***
 
+= 2.4.2 - xxx-xx-xx =
+* Fix - Action callback arguments count in ShipStation tracking integration #1841
+* Fix - Google Pay scripts loading on unrelated admin pages #1834
+* Fix - Do not ignore disabled APMs list in blocks #1865
+* Fix - Display Package Tracking metabox below Order actions when HPOS is active #1850
+* Fix - ApplePay use checkout form data to update shipping and billing #1832
+* Enhancement - Keep default "Place order" button text in continuation #1870
```

Comment:
It's a fix for another PR in this release, probably should not include to avoid confusion or instead of this line put the PR # after `#1840`